editor/components/block-switcher/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,16 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BlockSwitcher Test block switcher with blocks 1`] = `
+<Dropdown
+  className="editor-block-switcher"
+  contentClassName="editor-block-switcher__popover"
+  renderContent={[Function]}
+  renderToggle={[Function]}
+/>
+`;
+
+exports[`BlockSwitcher Test block switcher with multi block of different types. 1`] = `""`;
```

Comment:
According to this snapshots 3 tests don't render anything. It would read better to check against `null` in the test instead.

Reply:
Yup, I need to look back through this to update.